projects/packages/search/src/instant-search/lib/styles/_mixins.scss

```diff
@@ -109,14 +109,14 @@ $customberg-container-selector: '.jp-search-configure-app-wrapper' !default;
 	letter-spacing: inherit;
 }
 
-// !important is temporarily needed to override :not() selectors in certain themes like Twenty Twenty One
-@mixin inherit-parent-colors-important() {
-	background-color: inherit !important;
-	color: inherit !important;
+@mixin apply-default-overlay-colors() {
+	background-color: inherit;
+	color: #646970;
```

Comment:
The color look quite pale on dark theme,

<img width="209" alt="image" src="https://user-images.githubusercontent.com/1425433/186823169-a2d82ec0-3ced-4384-b96a-599421526880.png">

Compared to trunk:

<img width="198" alt="image" src="https://user-images.githubusercontent.com/1425433/186823229-6cad3395-5c9b-43ac-8165-42e6e54e4ddc.png">

Could we use different colors for dark/light theme?

Reply:
Recent changes to fix the button styling did not include the font colors/weights we originally had. I didn't see any comment that this was intentional though, so this change reintroduces the values we've had all along. The white in your screenshot has only been a part of the styling for about 2 weeks. Prior to that, it looked like it does in this branch.

That said, we could definitely go with white (or close to it) but we'd be breaking from the predefined theme colors. I'll have another look at this and see what I can come up with.

Reply:
After some more digging, I see we do have colours defined for these controls in our dark mode theme. They were just being overridden the same way the light colours were. I've added extra specificity to those rules so they should take effect correctly now. I'll add a screenshot to show the updated appearance.